site/docs/v1/tech/apis/_tenant-request-body.adoc

```diff
@@ -325,6 +325,24 @@ The password minimum age in seconds. When enabled FusionAuth will not allow a pa
 [field]#tenant.minimumPasswordAge.enabled# [type]#[Boolean]# [optional]#Optional# [default]#defaults to `false`# [since]#Available since 1.8.0#::
 Indicates that the minimum password age is enabled and being enforced.
 
+[field]#tenant.multiFactorConfiguration.authenticator.enabled# [type]#[Boolean]# [optional]#Optional# [default]#defaults to `true`# [since]#Available since 1.26.0#::
+When enabled, users may utilize an authenticator application to complete a two-factor authentication request. This is often referred to as TOTP (Time Based One Time Password), or a Google Authenticator.
```

Comment:
Is it `time-based`?

Reply:
"This is often referred to as TOTP (Time Based One Time Password), or a Google Authenticator." ->

"This method uses TOTP (Time-Based One-Time Password) as defined in [RFC 6238](https://tools.ietf.org/html/rfc6238) and often uses an native mobile app such as Google Authenticator."

Reply:
Yes, I like your version better. I had copy/pasted that from somewhere...maybe a tooltip?